docs/notes

```diff
@@ -115,6 +109,7 @@ reactor.run()
 ~~~~~~~~~~~~~~~~~~~~~~~
 
 //QUESTIONS//
-. what server architecture would this run on
-. what port on the server
-. how to determine internal sites. that technology?
+. how to determine internal ips. that technology?
+. I'm pretty sure I'm using virtualenv wrong...
```

Comment:
I'd like to chat to you about it's purpose. I'm using it rather sporadically, since I keep forgetting it's there...